tests/transform/test_rna_de_aggregate.py

```diff
@@ -0,0 +1,539 @@
+        datasets["test_data_mixed"] = mixed_data
+
+        # Transform data
+        output_data = transform_rna_de_aggregate(datasets=datasets)
+
+        # Should only contain mouse genes (ENSMUSG*), not human genes (ENSG*)
+        assert len(output_data) == 2  # Only 2 mouse genes
+        for entry in output_data:
+            assert entry["ensembl_gene_id"].startswith("ENSMUSG")
+
+    def test_transform_rna_de_aggregate_jax_tissue_mapping(self):
+        """Test that JAX models with 'Right Cerebral Hemisphere' tissue are mapped to 'Hemibrain'."""
+        # Create test data with JAX model and Right Cerebral Hemisphere tissue
+        jax_data = pd.DataFrame(
+            {
+                "ensembl_gene_id": ["ENSMUSG00000022892"],
+                "log2foldchange": [1.234567],
+                "padj": [0.001],
+                "model": ["JAX_model"],
+                "case": ["Tg"],
+                "control": ["Wt"],
+                "age": ["6 months"],
+                "sex": ["Female"],
+                "tissue": ["Right Cerebral Hemisphere"],
+            }
+        )
+
+        # Create JAX-specific datasets
+        datasets = {
+            "test_data_jax": jax_data,
+            "rnaseq_genotype_label_map": pd.DataFrame(
+                {
+                    "model": ["JAX_model"],
+                    "model_group": ["AD"],
+                    "display_label": ["JAX_model (Tg)"],
+                    "genotype": ["Tg"],
+                }
+            ),
+            "mouse_gene_metadata": pd.DataFrame(
+                {
+                    "ensembl_gene_id": ["ENSMUSG00000022892"],
+                    "gene_symbol": ["App"],
+                    "alias": [""],
+                }
+            ),
+            "model_info": pd.DataFrame(
+                {
+                    "model": ["JAX_model"],
+                    "matched_controls": ["JAX_model (Wt)"],
+                    "model_type": ["Transgenic"],
+                }
+            ),
+            "biodom_genes_mm": pd.DataFrame(
+                {
+                    "biodomain": ["Synaptic"],
+                    "abbr": ["Axon"],
+                    "label": ["Axon Function"],
+                    "color": ["#FF6B6B"],
+                    "go_id": ["GO:0030424"],
+                    "goterm_name": ["axon"],
+                    "n_symbol": [1],
+                    "symbol": ["App"],
+                    "ensembl_id": ["ENSMUSG00000022892"],
+                }
+            ),
+        }
+
+        # Transform data
+        output_data = transform_rna_de_aggregate(datasets=datasets)
+
+        # Should map tissue from "Right Cerebral Hemisphere" to "Hemibrain"
+        assert len(output_data) == 1
+        assert output_data[0]["tissue"] == "Hemibrain"
+        assert "jax" in output_data[0]["name"].lower()
+
+    def test_transform_rna_de_aggregate_age_sorting(self):
+        """Test that age entries are sorted numerically."""
```

Comment:
Same question w/ using synthetic data, but dropping in the age_data in place of something else.